app/src/main/kotlin/com/wire/android/ui/e2eiEnrollment/E2EIEnrollmentScreen.kt

```diff
@@ -227,6 +217,8 @@ private fun E2EIEnrollmentScreenContent(
     }
 }
 
+private const val E2EI_INFO_URL = "https://support.wire.com/hc/en-us/articles/9211300150685-End-to-end-identity"
```

Comment:
can we please have this link in the strings.xml file so it can be configured per app if there is a need to
e.g. https://github.com/wireapp/wire-android/blob/bd440050df34ec1a2f5e9704162ed2b653ce64bb/app/src/main/res/values/strings.xml#L266

Reply:
The link should be the non-localized version as well, so it is delegated to the browser locale as the rest of links from wire support.

https://support.wire.com/hc/articles/9211300150685